src/client/components/pages/entities/series.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2021  Akash Gupta
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import * as bootstrap from 'react-bootstrap';
+import * as entityHelper from '../../../helpers/entity';
+import {formatDate, getEntityKey, getEntityTable} from '../../../helpers/utils';
+import EntityAnnotation from './annotation';
+import EntityFooter from './footer';
+import EntityImage from './image';
+import EntityLinks from './links';
+import EntityRelatedCollections from './related-collections';
+import EntityTitle from './title';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+const {deletedEntityMessage, getTypeAttribute, getEntityUrl, ENTITY_TYPE_ICONS, getSortNameOfDefaultAlias} = entityHelper;
+const {Col, Row} = bootstrap;
+
+function SeriesAttributes({series}) {
+	if (series.deleted) {
+		return deletedEntityMessage;
+	}
+	const sortNameOfDefaultAlias = getSortNameOfDefaultAlias(series);
+	return (
+		<div>
+			<Row>
+				<Col md={3}>
+					<dl>
+						<dt>Sort Name</dt>
+						<dd>{sortNameOfDefaultAlias}</dd>
+					</dl>
+				</Col>
+				<Col md={3}>
+					<dl>
+						<dt>Series Type</dt>
+						<dd>{series.entityType}</dd>
+					</dl>
+				</Col>
+				<Col md={3}>
+					<dl>
+						<dt>Ordering Type</dt>
+						<dd>{series.seriesOrderingType.label}</dd>
+					</dl>
+				</Col>
+			</Row>
```

Comment:
do you mean to count the total number of items in the series and display in the frontend something like this:
```
<dl>
	<dt>Total Items</dt>
	<dd>{SomeValue}</dd>
</dl>
```

Reply:
Yes, that's what I'm talking about.
I'm not sure what the best way of counting the items is or where that code should live, but it might not be too hard to add.

What I originally had in mind was probably much too complicated (it involved a `count` property on the SeriesData ORM model…) but I think there should be an easy enough solution.

Reply:
Thanks for clarifying. I think we can simply use **seriesItems.length** to display the total number of items in the series. https://github.com/bookbrainz/bookbrainz-site/pull/649/files#diff-db9f5ec20bca1f2e5812e2fdd4066313ca3ed4dcfeb86ba21b055f1125143c3dR76